docs/software/installation.md

```diff
@@ -271,5 +271,118 @@ Once you managed to install Jam, make sure to understand how to use it.
 
 [:octicons-arrow-right-24: First Use][cheatsheet]
 
+### ... installing `jmwalletd`, `ob-watcher` and `jam` services (optional but recommended)
```

Comment:
maybe something like `configure to run services at start up` is a more clear description title?